src/main/java/com/saucelabs/ci/sauceconnect/AbstractSauceTunnelManager.java

```diff
@@ -533,6 +534,11 @@ public Process openConnection(
       final Process process =
           prepAndCreateProcess(
               username, apiKey, port, sauceConnectJar, options, printStream, sauceConnectPath);
+
+      // Print sauceconnect process stdout/stderr
+      new ProcessOutputPrinter(process.getInputStream(), (String x) -> logMessage(printStream, x)).start();
+      new ProcessOutputPrinter(process.getErrorStream(), (String x) -> logMessage(printStream, x)).start();
```

Comment:
Does it make sense to use different log levels for std out and std err: e.g. info and error?